### PR TITLE
Update buildscript

### PR DESCRIPTION
**Pull request by bombcar**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,5 +1,64 @@
-modGroup = bq_standard
-modVersion = 3.0.186-GTNH
-modBaseName = StandardExpansion
+modName = Standard Expansion
 
-forgeVersion = 1.7.10-10.13.4.1614-1.7.10
+# This is a case-sensitive string to identify your mod. Convention is to use lower case.
+modId = bq_standard
+
+modGroup =
```
Comment (SinTh0r4s) [inline gradle.properties]: A group in java is the common package(s) for a project. Here: `bq_standard`

File: dependencies.gradle [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// Add your dependencies here
+
+dependencies {
```
Comment (SinTh0r4s) [inline dependencies.gradle:3]: Add Better questing as GTNH dependency. It is already migrated. There are two versions of apis for this mod: `api` and `api2`. I am not sure which one you need to add, maybe both. Try try to add it as compileOnly first and see if it runs.

Comment (SinTh0r4s) [reply dependencies.gradle:3]: And you must remove the libs folder for that.

**Review by SinTh0r4s — CHANGES_REQUESTED** [review]

File: dependencies.gradle (hunk 2 above)
Comment (bombcar) [reply dependencies.gradle:3]: what is strange is nitpick can't find api2:

```
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not find BetterQuesting-master-SNAPSHOT-api2.jar (com.github.GTNewHorizons:BetterQuesting:master-SNAPSHOT:3.0.350-GTNH-g81d7f81-10).
     Searched in the following locations:
         https://jitpack.io/com/github/GTNewHorizons/BetterQuesting/master-SNAPSHOT/BetterQuesting-master-3.0.350-GTNH-g81d7f81-10-api2.jar
```

but :dev works fine

Comment (bombcar) [reply dependencies.gradle:3]: (and api doesn't work)

Comment (SinTh0r4s) [reply dependencies.gradle:3]: Need my pc for that. Might have time for it tomorrow

**Review by SinTh0r4s — CHANGES_REQUESTED** [review]
Same CI issue as in Backpacks

**Comment by SinTh0r4s** [comment]
The dependency to BetterQuesting should be defined as 
```
compileOnly("com.github.GTNewHorizons:BetterQuesting:master-SNAPSHOT:api")
compileOnly("com.github.GTNewHorizons:BetterQuesting:master-SNAPSHOT:api2")
```
but there is indeed an issue with the resolution of `master-SNAPSHOT` to an actual version with Jitpack. With fixed versions this would be fixed. Using `-dev` is fine for now. Also the code references CodeChickenLib, so it should be stated in dependencies.gradle as such. I'll push a reworked version.

File: gradle.properties [hunk 3]
```diff
@@ -1,5 +1,64 @@
...
+# that is annotated with @Mod) you want this to be true. When in doubt: leave it on false!
+containsMixinsAndOrCoreModOnly = false
+
+# If enabled, you may use 'shadowCompile' for dependencies. They will be integrated in your jar. It is your
+# responsibility check the licence and request permission for distribution, if required.
+usesShadowedDependencies = false
+
+# Optional parameter to customize the produced artifacts. Use this to preserver artifact naming when migrating older
+# projects. New projects should not use this parameter.
+#customArchiveBaseName =
```
Comment (SinTh0r4s) [inline gradle.properties]: The original build script named it `StandardExpansion`

**Review by SinTh0r4s — CHANGES_REQUESTED** [review]
Last one. Then it can be merged

**Comment by bombcar** [comment]
Went to compile for now as it won't load in CI without better question.

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**